src/main/java/seedu/address/logic/parser/LexSortCommandParser.java

```diff
@@ -8,24 +8,24 @@
 
 import java.util.List;
 
-import seedu.address.logic.commands.SortCommand;
+import seedu.address.logic.commands.LexSortCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
 
 /**
- * Parses input arguments and creates a new AttributeRemoveCommand object
+ * Parses input arguments and creates a new LexSortCommand object
  */
-public class SortCommandParser implements Parser<SortCommand> {
+public class LexSortCommandParser implements Parser<LexSortCommand> {
     public static final String MESSAGE_ONLY_ONE_PARAMETER =
-        "Note that this command accepts exactly ONE attribute as a parameter.";
+            "Note that this command accepts exactly ONE attribute as a parameter.";
     public static final String MESSAGE_EMPTY_ATTRIBUTE_NAME =
-        "The attribute name cannot be empty!";
+            "The attribute name cannot be empty!";
 
     private boolean isContainProhibitedCharacters(String str) {
         return str.chars().anyMatch(c -> PROHIBITED_CHARACTERS.chars().anyMatch(ng -> ng == c));
     }
 
     @Override
-    public SortCommand parse(String args) throws ParseException {
+    public LexSortCommand parse(String args) throws ParseException {
```

Comment:
Suggestion: how about creating a super class SortCommandParser, so that we can put together the common part of LexSortCommandParser::parse() and NumSortCommandParser::parse()?